TOOLS.rst

```diff
@@ -0,0 +1,201 @@
+This file explains the development tool chain around ixdat
+
+Tools used
+==========
+
+The following is a list of specific tools used during ixdat
+development:
+
+* **pytest** is used for running tests
+* **flake8** is used for linting
+* **sphinx** is used to build documentation
+
+The following is a list of "tool and commands runners":
+   
+* **invoke** is used during development, to run tools and other
+  pre-configured maintenance tasks inside the existing development
+  environment
+* **tox** is used as a tool to run all **QA** tools and commands
+  across all supported enviroments, typically before push or by
+  continuous integration (CI) tools
+
+Tool and command runners in more detail
+=======================================
+
+The reason there are two different tool runners is that they have very
+different purposes:
+
+**invoke** is used as general purpose, platform independent way of
```

Comment:
I'm missing some instructions, with the level of detail of the git hooks instructions, on how to get invoke working.
Naively checking out the [tools] branch, navigating the anaconda powershell prompt to the project folder, and typing 
`> invoke tox` 
gives the error
`invoke : The term 'invoke' is not recognized...`
I'm not completely surprised that something is missing, but I don't know what it is.

Reply:
Ah, yeah, of course:
```
> pip install invoke
> pip install tox
```
Works in anaconda prompt now but not git bash.

Reply:
Yeah. I added them all to `requirements-dev.txt` so `pip install -r requirements-dev.txt` in the anaconda prompt should get you everything including pytest, flake8, invoke and tox

Reply:
I added the following installation instruction section at the top of the file:

```rst
Install instructions
--------------------

Install all development tools by executing::

  pip install -r requirements-dev.txt

while being at the git archive root in your virtual environment or
anaconda environment.
```